xil_carma_cloud/cdasim_config/carma/lci_strategic_plugin/parameters.yaml

```diff
@@ -0,0 +1,50 @@
+# Double: multiplier to apply to the maximum allowable vehicle deceleration limit so we plan under our capabilities
+vehicle_decel_limit_multiplier : 1.0
+
+# Double: multiplier to apply to the maximum allowable vehicle acceleration limit so we plan under our capabilities
+vehicle_accel_limit_multiplier : 1.0
+
+# Double: The minimum distance in meters that the vehicle can be at before requiring a transition to the APPROACH state
+min_approach_distance : 20.0
+
+# Double: Downtrack distance until nearest intersection where the Trajectory Smoothing algorithm should activate
+trajectory_smoothing_activation_distance: 40.0
+
+# Double: A buffer infront of the stopping location which will still be considered a valid stop. Units in meters
+stopping_location_buffer : 4.0
+
+# Double: A buffer in seconds around the green phase which will reduce the phase length such that vehicle still considers it non-green
+green_light_time_buffer : 0.5
+
+# Double: Minimum allowable speed in trajectory smoothing algorithm m/s
+algo_minimum_speed : 0.894
```

Comment:
2mph is set by workzone. So in order to allow such low speed, we need to adjust the minimum_speed to match it so as not to use higher minimum_speed and not ignore the workzone.